cron_main.py

```diff
@@ -79,6 +80,7 @@
                                ('/tasks/math/enqueue/typeaheadcalc', TypeaheadCalcEnqueue),
                                ('/tasks/math/do/typeaheadcalc', TypeaheadCalcDo),
                                ('/tasks/posts/firebase_push', FirebasePushDo),
+                               ('/tasks/test_notification/([0-9]+)', TestNotificationController)
```

Comment:
In app.yaml:
```
- url: /tasks/.*
  script: cron_main.app
  login: admin
```

because we don't want to allow random people to run our tasks such as `/tasks/enqueue/usfirst_event_alliances/XXXX`